pkg/rpcclient/unwrap/unwrap.go

```diff
@@ -401,6 +401,9 @@ func checkResOK(r *result.Invoke, err error) error {
 	if r.State != vmstate.Halt.String() {
 		return fmt.Errorf("invocation failed: %s", r.FaultException)
 	}
+	if r.FaultException != "" {
+		return errors.New(r.FaultException)
```

Comment:
Not sure it is the best message, but `error: ` context seems excessive, FaultException usually contains something.

Reply:
`inconsistent result, HALTed with exception: %w`?

Reply:
@fyfyrchik, let's fix this comment and merge the PR. We'll remove stack marshalling error from `result.Invoke`'s FaultException in https://github.com/nspcc-dev/neo-go/issues/3359.

Reply:
fixed